docs/analyzer/user_guide.md

```diff
@@ -608,6 +619,11 @@ If you want to run the analysis with a specific compiler configuration
 instead of the auto-detection you can pass that to the
 `--compiler-info-file compiler_info.json` parameter.
 
+There are some implicit include paths (for example those which contain
+`include-fixed` directory) which are used only by GCC. By default CodeChecker
+doesn't collect them unless `--keep-gcc-fix-include` flag is given. For
```

Comment:
We collect (log) them we just do not use them for analysis.

Reply:
No, we don't log them. The implicit include paths are not logged to the compilation database.